### DISCUSSION TITLE
Add Fosscord support

### DISCUSSION THREAD

**Discussion by AruMoon** [42wim/matterbridge#1731] [New bridges suggestions]
**Describe the solution you'd like**
I wanted to have [Fosscord](https://fosscord.com) support on this bridge. This app has all the same api as Discord, just needed to specify instance endpoint instead of discord.com

**Comment by poVoq** [comment]
~~Fosscord is an Discord client~~. I think you misunderstand what Matterbridge does.

Edit: at least it used to. Well the question would be if they have the same webhook API (slack like).

Edit: According to their Roadmap the webhook support isn't finished yet: https://fosscord.com/roadmap/

**Comment by MathMan05** [comment]
sorry for the reply to a very old post, though it was never a client. Spacebar has a working webhook API, how hard would it be to bridge between spacebar and discord?
